openvpn/ironsocket/Argentina.ovpn

```diff
@@ -0,0 +1,196 @@
+setenv FORWARD_COMPATIBLE 1
+client
+push-peer-info
+server-poll-timeout 5
+dev tun
+dev-type tun
+nobind
+persist-key
+auth-user-pass
```

Comment:
Needs to be updated to read credentials from file, like this: https://github.com/haugene/docker-transmission-openvpn/blob/master/openvpn/pia/AU%20Melbourne.ovpn#L13